packages/core/src/components/widgets/SelectWidget.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { processSelectValue, WidgetProps } from "@rjsf/utils";
+
+function getValue(
+  event: React.SyntheticEvent<HTMLSelectElement>,
+  multiple: boolean
+) {
+  if (multiple) {
+    return Array.from((event.target as HTMLSelectElement).options)
+      .slice()
+      .filter((o) => o.selected)
+      .map((o) => o.value);
+  } else {
+    return (event.target as HTMLSelectElement).value;
+  }
+}
+
+function SelectWidget<T = any, F = any>({
```

Comment:
Can you add JSDoc here as well... take a look at how the other, already converted widgets are documented as a guide. Bonus points for documenting all the functions in this file.